src/dos/dos_keyboard_layout.h

```diff
@@ -0,0 +1,30 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2020-2021  The DOSBox Staging Team
```

Comment:
Because the content is identical, the copyright lines should all be left as-is.

The general rule is to keep a running history of authorship when notable additions or significant changes are made, in order of newest to oldest. 

So things like community authors or other teams (the DOSBox-X Team, etc)

For example, we merge the FluidSynth community patch from `Nikos Chantziaras` and from there we made many notable changes to the source from there:


``` text
/*
 *  SPDX-License-Identifier: GPL-2.0-or-later
 *
 *  Copyright (C) 2020-2021  The DOSBox Staging Team
 *  Copyright (C) 2020-2020  Nikos Chantziaras <realnc@gmail.com>
 *  Copyright (C) 2002-2011  The DOSBox Team
 *
```

Hope that helps!

Reply:
Oops, I grabbed the wrong copyright notice. Thanks for pointing that out.

(I hadn't checked for differences).